JASP-Desktop/analysis/boundqmlitem.cpp

```diff
@@ -21,7 +21,9 @@
 
 void BoundQMLItem::runRScript(const QString &script, bool whiteListedVersion)
 {
-	form()->runRScript(script, name(), whiteListedVersion);
+	QString id = parentListControl() ? (parentListControl()->name() + "." + parentListControlKey() + "." + name()) : name();
```

Comment:
I assume parentListControlKey() is always return something sensible whenever parentListControl() != nulltpr?

Reply:
Ooh yeah I see already